fplog/fplog_test.cpp

```diff
@@ -115,7 +118,9 @@ bool filter_test()
         fplog::write(msg);
     }
 
-    Lua_Filter* lua_filter = new Lua_Filter("lua_filter", "if fplog_message.text ~= nil and string.find(fplog_message.text, \"hello from Lua!\") ~= nil then filter_result = true else filter_result = false end");
+   // Lua_Filter* lua_filter = new Lua_Filter("lua_filter", "if fplog_message.text ~= nil and string.find(fplog_message.text, \"hello from Lua!\") ~= nil then filter_result = true else filter_result = false end");
```

Comment:
Please restore the original Lua filter here, you just broke this test, for your performance test to work you do not need to change filter here.